ssh_scan.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency('bindata', '~> 2.0')
   s.add_dependency('netaddr')
   s.add_dependency('net-ssh')
+  s.add_dependency('mongo')
```

Comment:
It's not a blocker just yet, but I'm starting to wonder how we'll want to deal with the converging dependencies that are being introduced by supporting multiple DB types.  I say this because we would want to avoid a situation where a user just wants ssh_scan binary, but they would need to install mongo/mysql/sqlite just to do that.

This is some food for thought, maybe we should discuss our options on the call.